exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Similar suggestion about scaffolding the observations; you could ask questions like "Would you expect Ideal diamonds to have the highest price?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>